tests/test_modeling_clip.py

```diff
@@ -556,6 +557,6 @@ def test_inference(self):
             torch.Size((inputs.input_ids.shape[0], inputs.pixel_values.shape[0])),
         )
 
-        expected_logits = torch.tensor([[24.5056, 18.8076]], device=torch_device)
+        expected_logits = torch.Tensor([[24.5701, 19.3049]]).to(torch_device)
```

Comment:
The new pre-preprocessing slightly changes the `logits`

Reply:
We shouldn't use torch.Tensor anymore, see #12027